modules/lvms-troubleshooting-performing-a-forced-cleanup.adoc

```diff
@@ -37,23 +41,23 @@ $ oc get logicalvolume
 No resources found
 ----
 
-.. If there are any `LogicalVolume` CRs remaining, remove their finalizers by running the following command:
+.. If the `LogicalVolume` CRs are present, remove their finalizers by running the following command:
 +
 [source,terminal]
 ----
 $ oc patch logicalvolume <name> -p '{"metadata":{"finalizers":[]}}' --type=merge <1>
 ----
-<1> Replace `<name>` with the name of the CR.
+<1> Replace `<name>` with the name of the `LogicalVolume` CR.
 
-.. After removing their finalizers, delete the CRs by running the following command:
+.. After removing the finalizers, delete the `LogicalVolume` CRs by running the following command:
```

Comment:
This is incorrect I believe, one first needs to trigger a delete of the LogicalVolumes, and then remove finalizers as necessary

@suleymanakbas91 please correct me if wrong here

Reply:
Updated the procedure as follows:

1. Check if the `LogicalVolume` custom resources (CRs) are present by running the following command:
+
[source,terminal]
----
$ oc get logicalvolume
----
+
.Example output
[source,terminal]
----
No resources found
----

a. If the `LogicalVolume` CRs are present, delete them by running the following command:
+
[source,terminal]
----
$ oc delete logicalvolume <name> <1>
----
<1> Replace `<name>` with the name of the `LogicalVolume` CR.

b. After deleting the `LogicalVolume` CRs, remove their finalizers by running the following command:
+
[source,terminal]
----
$ oc patch logicalvolume <name> -p '{"metadata":{"finalizers":[]}}' --type=merge <1>
----
<1> Replace `<name>` with the name of the `LogicalVolume` CR.

Reply:
@radeore - But I believe @jakobmoellerdev  had a different perspective on this, if I am not wrong.

Reply:
@sr1kar99 Oh right, my bad.  Removing finalizers step should be performed after deleting the resources.